src/app/app.component.html

```diff
@@ -16,13 +16,13 @@
           <mat-icon class="icon">{{ right.expanded ? 'fullscreen_exit' : 'fullscreen' }}</mat-icon>
         </button>
       </div>
-      <button (click)="viewer.toggleViewerVisible()">
-        Open tissue viewer
+      <button (click)="openViewer(viewer, 'https://entity-api.hubmapconsortium.org/entities/003d63df59a4975bbd680a1f6b706d67')">
```

Comment:
What is this url? Should probably go into environment if not a placeholder.

Reply:
Its a placeholder until its been merged with @adphillips2017 's changes.